package/gluon-web-private-wifi/luasrc/lib/gluon/config-mode/model/admin/privatewifi.lua

```diff
@@ -81,7 +81,7 @@ function f:write()
 				ssid       = ssid.data,
 				key        = key.data,
 				macaddr    = macaddr,
-				ifname     = suffix and 'wan' .. suffix,
+				ifname     = suffix and 'wan_wifi' .. suffix,
```

Comment:
Even though the package is called "private-wifi", we agreed some time ago to only continue using the term "WLAN", as "WiFi" is associated with the WiFi Alliance.

Reply:
How about making it a "wl" prefix, naming the interface "wlwanX"? Could also be used like "wlclient", "wlmesh", ... in the future as a common naming scheme.

Reply:
Sounds good, what about `wl_wanX` and `wl_client`/`wl_mesh` in the future?

Reply:
If we add a separator (which I do not have strong opinions on), I'd rather use `wl-` to match OpenWrt's `br-`.